### ISSUE TITLE
Add color to the interface

### ISSUE THREAD

**Issue by MartijnBraam** [budkin/jam#15]
Adding color to the interface will make some things slightly more easy to see at a glance like what panel is the active one. Like have the colors from the cmus interface.

**Comment by budkin** [comment]
I've been avoiding using colors in the interface, because they will probably not look good on windows (and someone has already forked my repo and added windows support, so windows and mac support is possible and planned). If I'm wrong and this is not an issue, I'll add them. I need to play with windows version first

**Comment by MartijnBraam** [comment]
You should detect the terminal type and use the corresponding color type, on windows you should avoid sending color escape sequences and under unix systems you can most likely always send the standard 16 color escape codes.

I don't know about Go but python has several libraries for handeling terminal colors based on the color support of the current terminal

**Comment by nlamirault** [comment]
I see in [micro repository][1]  some color schemes. It could be interesting to implements that.

[1]: https://github.com/zyedidia/micro/tree/master/runtime/colorschemes

**Comment by budkin** [comment]
So, I gave the windows version a shot and it fails to start with
`failed to create waveout: device ID out of range` 
@koron doesn't even let me submit an issue

**Comment by MartijnBraam** [comment]
That is an issue with the audio device, not with the colors so that is a seperate issue about windows support

**Comment by budkin** [comment]
Yeah, but if I had a chance to test windows version, I would have tested colors support as well
